requirements.txt

```diff
@@ -2,4 +2,4 @@ panel
 hvplot
 xarray
 netcdf4
-cartopy
+matplotlib
```

Comment:
I know this came up in the context of colour-maps, but I find it very strange - we don't actually use matplotlib, and it's a *big* requirement. Surely holoviews doesn't need matplotlib, when it can equally well plot with bokeh or plotly.